app/controllers/post_controller.py

```diff
@@ -26,3 +27,16 @@ def create_post(user_id: int) -> Any:
         post_service.create_post(post, file)
 
         return redirect(url_for("index_routes.index"))
+
+
+@post_routes.route('/delete', methods=['POST'])
+@session_required
+def delete_post(user_id) -> Any:
+    post_id = request.form.values()
```

Comment:
This might need to be refactored in the future depending on how the front end is handled (if csrf token is passed).

Reply:
Ack